pkg/gameserversets/controller_test.go

```diff
@@ -995,9 +996,15 @@ func createGameServers(gsSet *agonesv1.GameServerSet, size int) []agonesv1.GameS
 // newFakeController returns a controller, backed by the fake Clientset
 func newFakeController() (*Controller, agtesting.Mocks) {
 	m := agtesting.NewMocks()
-	wh := webhooks.NewWebHook(http.NewServeMux())
 	counter := gameservers.NewPerNodeCounter(m.KubeInformerFactory, m.AgonesInformerFactory)
-	c := NewController(wh, healthcheck.NewHandler(), counter, m.KubeClient, m.ExtClient, m.AgonesClient, m.AgonesInformerFactory)
+	c := NewController(healthcheck.NewHandler(), counter, m.KubeClient, m.ExtClient, m.AgonesClient, m.AgonesInformerFactory)
 	c.recorder = m.FakeRecorder
 	return c, m
 }
+
+// newFakeExtensions returns an extensions struct
+func newFakeExtensions() *Extensions {
+	ext := NewExtensions(webhooks.NewWebHook(http.NewServeMux()))
```

Comment:
One other thing I noticed - in each of these files, this can be simplified to a single line (removing the ext var and just doing `return NewExtensions(webhooks.NewWebHook(http.NewServeMux()))`.